include/wx/aui/auibar.h

```diff
@@ -343,6 +343,10 @@ class WXDLLIMPEXP_AUI wxAuiToolBarArt
     virtual int ShowDropDown(
                          wxWindow* wnd,
                          const wxAuiToolBarItemArray& items) = 0;
+    
+    //Provide opportunity for suclasses to recalculate colors
+    virtual void SysColourChanged() {}
```

Comment:
I don't really like this name, it seems weird to call this function from the ctor for example (because nothing has actually changed).

What about `SetColoursFromSystemTheme()` or maybe `UpdateColoursFromSystem()` or something like that?

Reply:
I'll update with UpdateColoursFromSystem()